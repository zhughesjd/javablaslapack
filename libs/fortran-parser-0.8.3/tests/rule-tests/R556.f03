

! Testing equivalence object list.
equivalence (a,b(3))
equivalence (x,y(2),z(4))
end
