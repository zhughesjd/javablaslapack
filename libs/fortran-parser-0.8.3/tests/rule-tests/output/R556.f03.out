********************************************
args[1]: R556.f03
R1101:main-program__begin:
R 555:equivalence-set-list__begin:
R 613:part-ref: id=[@7,50:50='a',<205>,4:13] hasSelectionSubscriptList=false hasImageSelector=false
R 612:data-ref: numPartRef=1
R 609:substring: hasSubstringRange=false
R 556:equivalence-object-list__begin:
R 619:section-subscript-list__begin:
R 406:int-literal-constant: digitString=[@11,54:54='3',<10>,4:17] kindParam=null
R 619:section-subscript: hasLowerBound=true hasUpperBound=false hasStride=false isAmbiguous=true
R 619:section-subscript-list: count=1
R 613:part-ref: id=[@9,52:52='b',<205>,4:15] hasSelectionSubscriptList=true hasImageSelector=false
R 612:data-ref: numPartRef=1
R 609:substring: hasSubstringRange=false
R 556:equivalence-object-list: count=1
R 555:equivalence-set-list: count=1
R 554:equivalence-stmt:
R 555:equivalence-set-list__begin:
R 613:part-ref: id=[@18,71:71='x',<205>,5:13] hasSelectionSubscriptList=false hasImageSelector=false
R 612:data-ref: numPartRef=1
R 609:substring: hasSubstringRange=false
R 556:equivalence-object-list__begin:
R 619:section-subscript-list__begin:
R 406:int-literal-constant: digitString=[@22,75:75='2',<10>,5:17] kindParam=null
R 619:section-subscript: hasLowerBound=true hasUpperBound=false hasStride=false isAmbiguous=true
R 619:section-subscript-list: count=1
R 613:part-ref: id=[@20,73:73='y',<205>,5:15] hasSelectionSubscriptList=true hasImageSelector=false
R 612:data-ref: numPartRef=1
R 609:substring: hasSubstringRange=false
R 619:section-subscript-list__begin:
R 406:int-literal-constant: digitString=[@27,80:80='4',<10>,5:22] kindParam=null
R 619:section-subscript: hasLowerBound=true hasUpperBound=false hasStride=false isAmbiguous=true
R 619:section-subscript-list: count=1
R 613:part-ref: id=[@25,78:78='z',<205>,5:20] hasSelectionSubscriptList=true hasImageSelector=false
R 612:data-ref: numPartRef=1
R 609:substring: hasSubstringRange=false
R 556:equivalence-object-list: count=2
R 555:equivalence-set-list: count=1
R 554:equivalence-stmt:
R9999:end-of-stmt:
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
Parser exiting normally
********************************************
